UI/Panels/InputConfigPanel.cs

```diff
@@ -42,6 +46,8 @@ public InputConfigPanel()
 
         void Init()
         {
+            inputsDataGridView.DataMember = null;
```

Comment:
If you data bind to a `DataSet` you also need to tell the control which table to use. That is the property `DataMember`.
If you data bind to a `DataTable` instead, you do not use the `DataMember` Property.

The trick to get the reset of the sorting working, is to directly data bind to the `DataTable`. Then for sorting the `DataTable.DefaultView` is used by the control, which can be accessed. Otherwise some internal Views are used which cannot be accessed.

The code switches from DataSet binding to direct DataTable binding.